flocker/ca/functional/test_validation.py

```diff
@@ -385,3 +389,6 @@ def test_requests_dns(self):
         the REST API when it is accessed via a domain name.
         """
         return self.assert_requests_validates(b"localhost", "control_dns")
+    test_requests_dns.skip = (
```

Comment:
:question: Can you merge the WIP on that issue and run it to see if this  proposed solution is valid?